processor/resourcedetectionprocessor/README.md

```diff
@@ -1,6 +1,6 @@
 # Resource Detection Processor
 
-Supported pipeline types: metrics, traces
+Supported pipeline types: metrics, traces, logs
 
 The resource detection processor can be used to detect resource information from the host,
 in a format that conforms to the [OpenTelemetry resource semantic conventions](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/resource/semantic_conventions/README.md), and append or
```

Comment:
Should update the comment below:

"in traces and metrics" -> "in telemetry data"